Resources/Prototypes/_ES/Fills/Crates/caches.yml

```
@@ -6,6 +7,46 @@
   suffix: ES
   abstract: true
 
+# Assassin
+- type: entity
+  parent: ESCrateCache
+  id: ESCrateCacheTraitorAssassin
+  suffix: ES, Assassin Cache
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: ESTraitorCacheAssassin
+
+- type: entityTable
+  id: ESTraitorCacheAssassin
+  table: !type:AllSelector
+    children:
+    - id: ESWeaponPistol9mmSilenced
+    - id: ESMagazine9mm
+      amount: 2
+    - id: ESSleepyPen
+      amount: 2
+      
+# Infiltrator
+- type: entity
+  parent: ESCrateCache
+  id: ESCrateCacheTraitorInfiltrator
+  suffix: ES, Infiltrator Cache
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: ESTraitorCacheInfiltrator
+
+- type: entityTable
+  id: ESTraitorCacheInfiltrator
+  table: !type:AllSelector
+    children:
+    - id: ClothingBackpackChameleon
```

```suggestion
    - id: ClothingBackpackChameleonFill
```
this is the unfilled variant. please test your PRs

---

also we probably need to do a pass on chameleon clothes so that they are properly limited to clothing that is actually accessible in-game

---

i did test it, i just didn't occur to me that i should probably look inside the chameleon bag.

also you are definitely right on the chameleon clothes, though, it's probably a conflict landmine so it's a bit scary.